modules/web/cypress/e2e/v2/stories/ssh-keys.spec.ts

```diff
@@ -87,6 +95,7 @@ describe('SSH Key Management Story', () => {
   });
 
   it('should delete the ssh key', () => {
+    Pages.Members.accessSideNavItem();
     Pages.SSHKeys.visit();
```

Comment:
We're already navigating to SSH Keys page using `Pages.SSHKeys.visit()` so why do we need `accessSideNavItem`?

Reply:
Test was failing as DDN menu was not opened and DOM node is not visible therefore fails.

Where as `Pages.SSHKeys.visit()` only clicks the button if DDN is opened.

reference test: `should create the ssh key`